banners/mobile/components/FullPageBanner.vue

```diff
@@ -6,7 +6,6 @@
 			</button>
 			<div class="wmde-banner-full-info">
 				<slot name="banner-text"/>
-				<slot name="progress"/>
```

Comment:
You could leave the slot in for adding it back later more easily, but removing it here probably saves a few bytes ...